Themes/Attributes/MyAccountAuthorizeAttribute.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using INZFS.Theme.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+
+namespace INZFS.Theme.Attributes
+{
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = true, Inherited = true)]
+    public class MyAccountAuthorizeAttribute : Attribute, IAuthorizeData
+    {
+        public string? Policy { get; set; }
```

Comment:
string is always nullable. Don't need a question mark here

Reply:
The question is not for making it nullable in this scenario.  its make it guarded parameterisable, Its a new feature in .net 5 (string immutability - remember that string is a reference type behaving like value type). 

There are some code references from .net code where Microsoft uses it. https://github.com/dotnet/aspnetcore/blob/main/src/Security/Authorization/Core/src/AuthorizeAttribute.cs

Please let me know if this passes it - if not, I'll remove the "?" mark